apps/ensadmin/src/app/gql/ponder/page.tsx

```diff
@@ -8,15 +8,8 @@ type PageProps = {
 };
 
 export default async function PonderGraphQLPage({ searchParams }: PageProps) {
-  const { ensnode = defaultEnsNodeUrl() } = await searchParams;
+  const ensNodeUrl = selectedEnsNodeUrl(parseNextJsPageSearchParams(await searchParams));
+  const ensNodePonderApiUrl = new URL(`/ponder`, ensNodeUrl);
 
-  const baseUrl = Array.isArray(ensnode)
-    ? ensnode[0]
-    : typeof ensnode === "string"
-      ? ensnode
-      : defaultEnsNodeUrl();
-
-  const url = new URL(`/ponder`, baseUrl).toString();
-
-  return <GraphiQLEditor url={url} />;
+  return <GraphiQLEditor graphQlApiUrl={ensNodePonderApiUrl.toString()} />;
```

Comment:
If I am to be pedantic, I'm not sure why we need to prefix this url with `graphQlApi` since graphiQL is a GraphQL UI, the context is already that the URL is for GraphQL API URLs only.

Reply:
Right, will take care of that 👍